core/src/core-api/dom-elements.js

```diff
@@ -0,0 +1,55 @@
+/** @namespace */
+class LuigiElementsManager {
+  /**
+   * Use the functions to get important DOM elements.
+   * @name LuigiElements
+   */
+  /**
+   * Returns the shellbar component.
+   * @returns {Object} the shellbar DOM element.
+   * @memberof LuigiElements
+   */
+  getShellbar() {
+    return document.getElementsByClassName('fd-shellbar')[0];
+  }
+
+  /**
+   * Returns the shellbar actions component.
+   * @returns {Object} the shellbar actions DOM element.
+   * @memberof LuigiElements
+   */
+  getShellbarActions() {
+    return document.getElementsByClassName('fd-shellbar__actions')[0];
+  }
+
+  /**
+   * Returns all microfrontend iframes including iframe from the modal if it exists.
+   * @returns {Object} an array of all microfrontend iframes from the DOM.
+   * @memberof LuigiElements
+   */
+  getMicrofrontendIframes() {
+    return [...document.querySelectorAll('.iframeContainer iframe')].concat([
+      ...document.querySelectorAll('.iframeModalCtn iframe')
+    ]);
+  }
+
+  /**
+   * Returns the active microfrontend iframe. If there is a modal, which includes the microfrontend iframe, the function will return this iframe.
```

Comment:
Please add a line break to make it easier readable without scrolling.

Reply:
Done